public/app/core/components/TimePicker/TimePickerWithHistory.test.tsx

```diff
@@ -116,11 +116,11 @@ describe('TimePickerWithHistory', () => {
     await userEvent.click(screen.getByLabelText(/Time range selected/));
 
     for (const [inputFrom, inputTo] of inputRanges) {
-      await userEvent.click(screen.getByLabelText(/Time range selected/));
```

Comment:
minor nit: think it makes more sense to leave this where it is and remove the click outside the loop on line 116 instead? 🤔